components/style/animation.rs

```diff
@@ -825,6 +825,10 @@ fn interpolate_transform_list(from_list: &[TransformOperation],
                 }
                 (&TransformOperation::Rotate(fx, fy, fz, fa),
                  &TransformOperation::Rotate(tx, ty, tz, ta)) => {
+                    let norm_f = ((fx * fx) + (fy * fy) + (fz * fz)).sqrt();
+                    let norm_t = ((tx * tx) + (ty * ty) + (tz * tz)).sqrt();
+                    let (fx, fy, fz) = (fx / norm_f, fy / norm_f, fz / norm_f);
+                    let (tx, ty, tz) = (tx / norm_t, ty / norm_t, tz / norm_t);
                     if fx == tx && fy == ty && fz == tz {
                         let ia = fa.interpolate(&ta, time).unwrap();
                         result.push(TransformOperation::Rotate(fx, fy, fz, ia));
```

Comment:
I wonder if the spec says anything about what the interpolated values for the direction vector should be...script can see this, right? Worth checking what other browsers do and adding a test if you're feeling particularly pedantic